src/components/Posts/Posts.js

```diff
@@ -17,14 +17,20 @@ function Posts({ posts, intro, id }) {
               id={`post-${post.id}`}
             >
               <div className={styles.card}>
-                <FeaturedImage
-                  className={styles.image}
-                  image={post?.featuredImage?.node}
-                  width={340}
-                  height={340}
-                />
+                <Link href={post?.uri ?? '#'}>
+                  <a className={styles[`image-holder`]}>
```

Comment:
Taking the image out of the tabindex prevents non-sighted users from hearing the link announced twice.
```suggestion
                  <a className={styles[`image-holder`]} tabindex="-1">
```

Reply:
🔥